test/harnesses/StakerHarness.sol

```diff
@@ -1,17 +1,17 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 pragma solidity ^0.8.23;
 
-import {DelegationSurrogateVotes} from "src/DelegationSurrogateVotes.sol";
-import {Staker} from "src/Staker.sol";
-import {StakerPermitAndStake} from "src/extensions/StakerPermitAndStake.sol";
-import {StakerOnBehalf} from "src/extensions/StakerOnBehalf.sol";
-import {StakerDelegateSurrogateVotes} from "src/extensions/StakerDelegateSurrogateVotes.sol";
+import {DelegationSurrogateVotes} from "../../src/DelegationSurrogateVotes.sol";
```

Comment:
This import looks unneeded, actually. Not introduced in this PR, but while we're updating imports, let's go ahead and remove it.